pkg/ddl/table.go

```diff
@@ -575,6 +568,11 @@ func (w *worker) recoverTable(t *meta.Meta, job *model.Job, recoverInfo *Recover
 	if err != nil {
 		return ver, errors.Trace(err)
 	}
+	err = clearTablePlacementAndBundles(w.ctx, recoverInfo.TableInfo)
+	if err != nil {
+		return ver, errors.Trace(err)
+	}
```

Comment:
This function sends a HTTP request. If there is a performance issue for `onRecoverSchema`, we can solve it by batching bundles from all involved `tableInfo`.